redash/handlers/api.py

```diff
@@ -197,6 +198,7 @@ def json_representation(data, code, headers=None):
 
 api.add_org_resource(QueryTagsResource, "/api/queries/tags", endpoint="query_tags")
 api.add_org_resource(DashboardTagsResource, "/api/dashboards/tags", endpoint="dashboard_tags")
+api.add_org_resource(metrWidgetTagsResource, "/api/metrwidgets/<widget_id>/tags", endpoint="widgets_tags")
```

Comment:
For the sake of pep8, the class name must start with a capital letter. Maybe too much javascript/python context change? hahaha

Reply:
thanks yeah it was js effect :D , please find update[ in this commit](https://github.com/metr-systems/redash/pull/4/commits/8c7f5078b424aa0f46ff997bcfd71670907c13dd)